client/app/telecom/telephony/alias/special/rsva/telecom-telephony-alias-special-rsva.controller.js

```diff
@@ -0,0 +1,243 @@
+angular.module('managerApp').controller('TelecomTelephonyAliasSpecialRsvaCtrl', class TelecomTelephonyAliasSpecialRsvaCtrl {
+  constructor(
+    $filter, $q, $stateParams, $translate,
+    costs, OvhApiOrder, OvhApiTelephony, OvhApiTelephonyService, telephonyBulk, Toast, URLS,
+  ) {
+    this.$filter = $filter;
+    this.$q = $q;
+    this.serviceName = $stateParams.serviceName;
+    this.billingAccount = $stateParams.billingAccount;
+    this.$translate = $translate;
+    this.costs = costs;
+    this.OvhApiOrder = OvhApiOrder;
+    this.OvhApiTelephony = OvhApiTelephony;
+    this.OvhApiTelephonyService = OvhApiTelephonyService;
+    this.telephonyBulk = telephonyBulk;
+    this.Toast = Toast;
+    this.URLS = URLS;
+  }
+
+  $onInit() {
+    this.rsva = {};
+    this.rsvaForm = {};
+
+    this.links = _.pick(this.URLS, ['transferTable', 'deontology', 'graphicCharter', 'svaToSvaPlus']);
+    this.tariffBearingPrice = `${this.costs.rsva.tariffBearing.value} ${(this.costs.rsva.tariffBearing.currencyCode === 'EUR' ? '€' : this.costs.rsva.tariffBearing.currencyCode)}`;
+
+    this.isLoading = true;
+
+    // Initialize bulk datas
+    this.bulkDatas = {
```

Comment:
**datum** (neuter, second declension)  

Case | Singular | Plural
-----|----------|--------
nominative | datum | **data**
genitive | datī | datōrum
dative | datō | datīs
accusative | datum | data
ablative | datō | datīs
vocative | datum | data
  
Just joking :stuck_out_tongue:

Reply:
In all seriousness, it's a nice thing to know to name variable:

`data.foreach(datum => console.log(datum))`

Although, to me, naming a variable `data` is already a bad practice (because it doesn't actually say what's the content of the variable... All variables contain data)